plugins/modules/zos_copy.py

```diff
@@ -2449,7 +2452,7 @@ def run_module(module, arg_def):
 
     try:
         if not is_uss:
-            res_args["changed"] = allocate_destination_data_set(
+            res_args["changed"], res_args["dynamic_values_dest"] = allocate_destination_data_set(
```

Comment:
I think we could use "dest_data_set_attrs" or similar instead of "dynamic_values_dest". I get that those are the values created dynamically but at the end of the day they're just the attributes of the data set. Same for the action plugin variable name.